chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -111,15 +109,22 @@ def _is_chainerx_supported(self, input_arrays):
         return True
 
     def forward_chainerx(self, inputs):
-        # TODO(niboshi): Current implementation is only intended to support
-        # MNIST example.
         x, t = inputs
         n_classes = x.shape[1]
         score = chainerx.log_softmax(x, axis=1)
         mask = (t[:, chainerx.newaxis] == chainerx.arange(
             n_classes, dtype=t.dtype, device=x.device)).astype(score.dtype)
-        # TODO(beam2d): implement mean
-        y = -(score * mask).sum() * (1 / x.shape[0])
+
+        neg_log_p = -score * mask
+        if self.reduce == 'mean':
+            if self.normalize:
+                count = mask.sum()
+            else:
+                count = x.shape[0]
+            y = neg_log_p.sum() * (1 / count)
```

Comment:
Could you fix the test failure on this line? It's due to the fact that https://github.com/chainer/chainer/pull/7075 was never backported. As `count` doesn't participate in the graph, you could just in the `if self.normalize:` path convert the `count` to a Python primitive number. Changing the `*` with `/` might be fine as well.
```
TypeError: unsupported operand type(s) for /: 'int' and 'chainerx.ndarray'
```

Reply:
Fixed!